src/views/HomeView.vue

```diff
@@ -41,7 +41,78 @@
           Mayo Clinic - Cardiomyopathy</a>
       </li>
     </ul>
+    <h5>Latests News/Journals </h5>
+    <div>
+      <div class="row">
+        <div v-for="card in cards" :key="card.id" class="col-sm-3">
+          <div class="card" @click="golink(card.link)">
+            <div class="card-body">
+              <h5 class="card-title">
+                {{ card.title }}
+              </h5>
+              <p class="card-text">
+                {{ card.description }}
+              </p>
+              <p class="card-text">
+                {{ card.date }}
+              </p>
+            </div>
+          </div>
```

Comment:
Is it possible to put the card as a component?

Reply:
Not feasible with the time left on the project.